gmail-gcalender-scheduling-agent/calendar_api.py

```
@@ -0,0 +1,99 @@
+            "maxResults": max_results,
+        }
+    ) or []
+
+def create_event(identifier: str, calendar_id: str, event: Dict[str, Any]) -> Dict:
+    start_dt = ((event.get("start") or {}).get("dateTime")) or event.get("start_datetime")
+    end_dt   = ((event.get("end") or {}).get("dateTime")) or event.get("end_datetime")
+    tz       = ((event.get("start") or {}).get("timeZone")
+                or (event.get("end") or {}).get("timeZone")
+                or event.get("time_zone"))
+    if not start_dt or not end_dt:
+        raise ValueError("create_event: start/end datetime missing")
```

Ruff (0.14.2)

Avoid specifying long messages outside the exception class

(TRY003)